app/src/main/res/values/styles.xml

```diff
@@ -29,6 +29,16 @@
         <item name="android:textColor">@color/colorPrimary</item>
     </style>
 
+    <style name="AppText.StatusHeader" parent="AppText">
```

Comment:
Jeśli mogę dorzucić swoje trzy grosze:
Nie ma potrzeby definiować `parent="AppText"`. Zbędny szum. `AppText.StatusHeader` z automatu dziedziczy po `AppText` poprzez "notację z kropką" (dot notation inheritance). 

(_Uwaga tyczy się w sumie większości stylów w tym plików. Można to trochę wyczyścić..._).

Reply:
Dzięki, zrobię z tym porządek przy kolejnym PR.